python_modules/dagster/dagster/daemon/daemon.py

```diff
@@ -22,7 +23,12 @@ def get_default_daemon_logger(daemon_name):
 
 
 DAEMON_HEARTBEAT_ERROR_LIMIT = 5  # Show at most 5 errors
-TELEMETRY_LOGGING_INTERVAL = 3600  # Interval (in seconds) at which to log that daemon is alive
+TELEMETRY_LOGGING_INTERVAL = 3600 * 24  # Interval (in seconds) at which to log that daemon is alive
+_telemetry_daemon_session_id = str(uuid.uuid4())
+
+
+def get_telemetry_daemon_session_id() -> str:
+    return _telemetry_daemon_session_id
```

Comment:
this is sort of dubious since its always set [1] 

also `uuid` generation isnt free so i dont love doing it on every process that imports dagster

Reply:
uuuh this stuff is all already landed